vault/resource_jwt_auth_backend_role.go

```diff
@@ -406,6 +406,8 @@ func jwtAuthBackendRoleDataToWrite(d *schema.ResourceData, create bool) map[stri
 			boundClaims[key] = claims
 		}
 		data["bound_claims"] = boundClaims
+	} else {
+		data["bound_claims"] = make(map[string]interface{}, 0)
```

Comment:
The empty map is defined on line [396](https://github.com/hashicorp/terraform-provider-vault/pull/1469/files#diff-4ce5833472afbb5d5a963d14f398fcba1ff7736c48af6b03f49cf3bca0824537R396) above. If we move that before the conditional we can drop the `else{}` and always set `bound_claims`

Reply:
I've done this.